Sources/TabBar/View/TabBar.swift

```diff
@@ -23,9 +23,15 @@
 
 import SwiftUI
 
+public enum TabBarVisibility: CaseIterable {
+  case visible,
+       invisible
+}
```

Comment:
Move the declaration of this enum to its own file called `TabBarVisibility.swift` and put it in the `Preferences` folder. Use tab of size 4 and do not use comma to declare multiple cases, one `case` keyword for each